src/deadline/job_attachments/upload.py

```diff
@@ -191,7 +191,7 @@ def upload_assets(
 
         if manifest_write_dir:
             self._write_local_manifest(
-                manifest_write_dir, manifest_name, full_manifest_key, manifest
+                manifest_write_dir, manifest_name, full_manifest_key, manifest, None
```

Comment:
`_write_local_manifest` already defaults `root_dir` to `None`, it doesn't need to be explicitly passed in.

https://github.com/stangch/deadline-cloud-stangch/blob/d69b729ea66a6475b049718bb00d6f41d3249ddd/src/deadline/job_attachments/upload.py#L239